### PR TITLE
[14.0][ADD] sale_pricelist_discount_decimal_precision

### PR DESCRIPTION
**Pull request by solo4games**
ADD  sale_pricelist_discount_decimal_precision.
This module enables the use of discount decimal precision for pricelist items in Odoo

### CODE REVIEW

File: sale_pricelist_discount_decimal_precision/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Ooops Discount Decimals",
```
Comment (Aldeigja) [inline sale_pricelist_discount_decimal_precision/__manifest__.py]: Module name should be "Sale Pricelist Discount Decimal Precision"

File: sale_pricelist_discount_decimal_precision/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "summary": "Use Discount decimal precision for pricelist items",
+    "version": "14.0.1.0.1",
+    "category": "Productivity",
```
Comment (Aldeigja) [inline sale_pricelist_discount_decimal_precision/__manifest__.py]: "Sale" category fits better here.

**Review by Aldeigja — CHANGES_REQUESTED** [review]

File: sale_pricelist_discount_decimal_precision/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+* Ilyas <https://ooops404.com>
```
Comment (francesco-ooops) [inline sale_pricelist_discount_decimal_precision/readme/CONTRIBUTORS.rst]: * Ooops404 <https://ooops404.com>

  * Ilyas <irazor147@gmail.com>
  * Your contact (email optional)

File: sale_pricelist_discount_decimal_precision/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Once installed, the module will add two new fields to pricelist items:
+percent_price and price_discount, both of which accept discount precision.
```
Comment (francesco-ooops) [inline sale_pricelist_discount_decimal_precision/readme/USAGE.rst]: Remove and change to: 

Go to Technical > Database Structure > Decimal Accuracy
Change the number of decimals for "Discount"
Pricelist will now accept discounts with the set number of decimals

**Review by francesco-ooops — APPROVED** [review]
Minor changes, functionally LGTM

**Review by Aldeigja — APPROVED** [review]
Code ok.

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers could you review?

**Comment by francesco-ooops** [comment]
@pedrobaeza what do you think?

**Pull request merged**